src/app/components/mnoe-organizations-list/mnoe-organizations-list.coffee

```diff
@@ -24,6 +24,7 @@
     # table generation - need to get the locale first
     $translate(
       ["mnoe_admin_panel.dashboard.organization.account_frozen_state",
+      "mnoe_admin_panel.dashboard.organization.demo_account_state",
```

Comment:
Do you need to translate `mnoe_admin_panel.dashboard.organization.demo_account_state` here. It is not fetched from `locale` array but passed to the translate `directive`.